vllm/model_executor/models/olmo.py

```diff
@@ -61,7 +61,9 @@
     hf_model_weights_iterator,
 )
 from vllm.sequence import SamplerOutput
-from vllm.transformers_utils.configs.olmo import OLMoConfig
+
+# this model must need this dependency
+from hf_olmo import OLMoConfig
```

Comment:
What does this mean? Why is `hf_olmo` required?

Reply:
`OLMo` model use this dependency in the configuration file https://huggingface.co/allenai/OLMo-1B/blob/main/configuration_olmo.py. And this will cause error when loading `config.json` file. https://buildkite.com/vllm/ci/builds/1707#018de7af-2b65-4c56-bbdc-f9db454e2cc5 So we can safely use this in this model.

Reply:
Got it. Thanks for the clarification!